src/app/loadout-builder/LoadoutBuilder.tsx

```diff
@@ -375,16 +375,14 @@ function LoadoutBuilder({
           <GeneratedSets
             sets={filteredSets}
             lockedMap={lockedMap}
+            lockedMods={lockedMods}
```

Comment:
uhhh, might have some merge conflicts incoming

Reply:
Yeah that is cool, get your stuff in and I will sort them out.